hiroaki-android/build.gradle

```diff
@@ -43,8 +43,7 @@ android {
 }
 
 dependencies {
-    // api "me.jorgecastillo:hiroaki-core:0.0.7"
-    api project(":hiroaki-core")
+    api "me.jorgecastillo:hiroaki-core:0.0.8"
```

Comment:
I tried doing a `debugApi project(":hiroaki-core")`, and only do `releaseApi "me.jorgecastillo:hiroaki-core:0.0.8"`, which lets you make sure tests pass, but won't let you do release builds unless the release core version is available.